src/SeoToolkit.Umbraco.ScriptManager/wwwroot/ScriptManager/lang/nl-nl.xml

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8" standalone="yes"?>
+<language alias="nl">
+	<area alias="siteManager">
+		<key alias="create">Script Aanmaken</key>
```

Comment:
And a small remark, I think "Aanmaken" shouldn't start with a capital letter, same for "Audit Aanmaken"